### DISCUSSION TITLE
Quicker GPS fix

### DISCUSSION THREAD

**Discussion by BobSquarePants** [organicmaps/organicmaps#8606] [Ideas]
Hi,

Time to time it might take time to have a GPS fix..

Would it be possible to manually set the actual position on the map, to assist the position acquisition ?

Thanks.

**Comment by biodranik** [comment]
Selecting position manually won't speed up the GPS fix, because it depends on a system, chip, and satellites signal strength, not on the application.

**Reply by BobSquarePants** [reply]
Thanks @biodranik , humm to bad then in Android system GPS fix, we can provide a ~ lat, lon coordinate to speed-up the fix :/
I will suggest them..
